guides/image-sources/prov-create-host-pxe.iuml

```diff
@@ -0,0 +1,16 @@
+User -> Foreman : clicks **Create Host**
+Foreman -> Proxy : requests free IP address
+Proxy -> DHCP : reads DHCP cofiguration
+note over Foreman : pings IP address
+User -> Foreman : submits the **Create host** form
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : renders bootloader configuration files
+end
+Proxy -> TFTP : deploys bootloader configuration files
+Foreman -> Proxy : commands to download installation media
+note over Proxy : downloads installation media
```

Comment:
They are not a section delimiter. They are describing behavior that is happening on the participant without collaboration with other participants on the diagram. It's like "operation on the self".

Reply:
Btw, I can change the representation of these self operations to the following instead of notes:
```
Participant -> Participant : action
```
Would that be better?

Reply:
@stejskalleos Hello? :) ^^

Reply:
Oh, sorry missed that one; I would personally prefer the variant with action:
```
Foreman -> Foreman : action
```